tensorflow/go/genop/generate.sh

```diff
@@ -46,8 +46,8 @@ fi
 # Ensure that protoc-gen-go is available in $PATH
 # Since ${PROTOC} will require it.
 export PATH=$PATH:${GOPATH}/bin
-mkdir -p ./internal/proto
+mkdir -p ../pb
```

Comment:
It might be confusing to have `genop/generate.sh` generate files in `../pb`. Either we should have a `pb` directory that contains the script to generate the protos, or we keep the `pb` directory inside `genop`.